src/DynamoCoreWpf/Utilities/MessageBoxUtilities.cs

```diff
@@ -29,6 +30,10 @@ MessageBoxResult IMessageBox.Show(string msg, string title, bool showRichTextBox
             {
                 return DynamoMessageBox.Show(msg, title, showRichTextBox, button, img);
             }
+            MessageBoxResult IMessageBox.Show(Window owner, string msg, string title, bool showRichTextBox, MessageBoxButton button, MessageBoxImage img)
+            {
+                return DynamoMessageBox.Show(owner,msg, title, showRichTextBox, button, img);
+            }
```

Comment:
Why do we need this?

Reply:
for both the `owner` and `showRichTextBox` param.
Since this type and interface are really `internal` we can probably just delete all these overloads and simplify whenever we want.

Reply:
ugh well, just to correct this a bit - 
the interface is internal, but the `MessageBoxService` and these Show methods implementing that interface are public - though the whole thing is in DynamoCoreWPF.dll so I think it requires just a bit more thinking before we refactor this.